shared/health/src/main/java/io/pravega/shared/health/impl/HealthEndpointImpl.java

```diff
@@ -62,12 +63,17 @@ public String getDefaultContributorName() {
     @NonNull
     @Override
     public Health getHealth(String id) {
+        Health health;
         if (id == null || id.equals(root.getName())) {
-            return updater.getLatestHealth();
+            health = updater.getLatestHealth();
```

Comment:
I don't see the actual health checks implemented in this PR. (_A class that implements AbstractHealthContributor_ in Controller and Segment Store) . Is that going to be part of another PR?

Reply:
That's correct. Those are intended to be added gradually.

Reply:
Okay